14.serial_search/serial_search.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'date'
+
+def serial_search(input)
+  year = 1900 + (input[0] + input[4]).to_i
+  days = input[1..3].to_i
+  production = input[5..7].to_i
+
+  first_date = Date.new(year)
+  manufactured = first_date + (days - 1)
+
+  factory, order = (1..499).cover?(production) ? ['カラマズー', production] : ['ナッシュビル', production - 500]
```

Comment:
```ruby
NASHVILLE_SERIAL_START = 500
```

みたいな定数を作って、

```suggestion
  factory, order = production < NASHVILLE_SERIAL_START ? ['カラマズー', production] : ['ナッシュビル', production - NASHVILLE_SERIAL_START]
```

にすると、ちゃんと考えている感が出そう